### PR TITLE
docs: remove demo-app docs from levski

### PR DESCRIPTION
**Pull request by ajcasagrande**
**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [X] I am not introducing a new dependency (add notes below if you are)
- [N/A] I have added unit tests for the new feature or bug fix (if not, why?)
- [N/A] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [N/A] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,8 +39,6 @@ The figure below illustrates the software flow through the architecture componen
 
 # Getting Started
 
-Learn how to configure and run the service by following these [instructions](./doc/setup.md).
```
Comment (ChristianDarr-personal) [inline README.md:42]: If we are removing the link to the `setup.md` do we want to just get rid of the repetitive files themselves? I.e. `setup.md`, `custom-build.md`, `general-usage.md`, `running-guide.md`. Or is that a discussion for later when we potentially move the docs?

**Review by ChristianDarr-personal — COMMENTED** [review]
LGTM just left one general comment about docs structure.

**Review by presatish — CHANGES_REQUESTED** [review]
I see README.md file referred in many places and the links to it are broken for some reason. For e.g. https://github.com/EdgeX-Camera-Management/device-onvif-camera/blob/remove-demo-app-docs-levski/doc/general-usage.md#next-steps

**Review by presatish — COMMENTED** [review]
Can we also move the `build-nats` stuff to a different page as we have in main branch for levski too?

**Comment by presatish** [comment]
> Can we also move the `build-nats` stuff to a different page as we have in main branch for levski too?

Never mind I see another PR for this now.

File: README.md (hunk 1 above)
Comment (presatish) [reply README.md:42]: For now we can keep these files until the migration is done for Edgex docs.

**Review by presatish — APPROVED** [review]

**Pull request merged**